docs/website/docs/general-usage/incremental-loading.md

```diff
@@ -482,38 +451,28 @@ def repo_issues(
+        # Last_value is updated after every page
         print(updated_at.last_value)
 ```
 
-Here we add `updated_at` argument that will receive incremental state, initialized to
-`1970-01-01T00:00:00Z`. It is configured to track `updated_at` field in issues yielded by
-`repo_issues` resource. It will store the newest `updated_at` value in `dlt`
-[state](state.md) and make it available in `updated_at.start_value` on next pipeline
-run. This value is inserted in `_get_issues_page` function into request query param **since** to [Github API](https://docs.github.com/en/rest/issues/issues?#list-repository-issues)
-
-In essence, `dlt.sources.incremental` instance above
-* **updated_at.initial_value** which is always equal to "1970-01-01T00:00:00Z" passed in constructor
-* **updated_at.start_value** a maximum `updated_at` value from the previous run or the **initial_value** on first run
-* **updated_at.last_value** a "real time" `updated_at` value updated with each yielded item or page. before first yield it equals **start_value**
-* **updated_at.end_value** (here not used) [marking end of backfill range](#using-end_value-for-backfill)
+Here we add an `updated_at` argument that will receive incremental state, initialized to `1970-01-01T00:00:00Z`. It is configured to track the `updated_at` field in issues yielded by the `repo_issues` resource. It will store the newest `updated_at` value in `dlt` [state](state.md) and make it available in `updated_at.start_value` on the next pipeline run. This value is inserted in the `_get_issues_page` function into the request query param **since** to the [GitHub API](https://docs.github.com/en/rest/issues/issues?#list-repository-issues).
 
-When paginating you probably need **start_value** which does not change during the execution of the resource, however
-most paginators will return a **next page** link which you should use.
+In essence, the `dlt.sources.incremental` instance above:
+* **updated_at.initial_value** which is always equal to "1970-01-01T00:00:00Z" passed in the constructor
+* **updated_at.start_value** a maximum `updated_at` value from the previous run or the **initial_value** on the first run
+* **updated_at.last_value** a "real-time" `updated_at` value updated with each yielded item or page. Before the first yield, it equals **start_value**
+* **updated_at.end_value** (here not used) [marking the end of the backfill range](#using-end_value-for-backfill)
 
-Behind the scenes, `dlt` will deduplicate the results ie. in case the last issue is returned again
-(`updated_at` filter is inclusive) and skip already loaded ones.
+When paginating, you probably need the **start_value** which does not change during the execution of the resource, however, most paginators will return a **next page** link which you should use.
 
+Behind the scenes, `dlt` will deduplicate the results, i.e., in case the last issue is returned again (`updated_at` filter is inclusive) and skip already loaded ones.
```

Comment:
```suggestion
Behind the scenes, dlt will deduplicate the results, i.e., in case the last issue is returned again (`updated_at` filter is inclusive) and skip already loaded ones.
```